app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(42, 3)) == 42
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    parts = split_integer(42, 7)
+    assert all(part == parts[0] for part in parts)
```

Comment:
you can simplify the test, just write what you expect to receive after execution this function

Reply:
is it a bad practise to make such test? just curious